light_curves/code_src/wise_functions.py

```diff
@@ -51,19 +84,44 @@ def wise_get_lightcurves(sample_table, *, radius=1.0, bandlist=["W1", "W2"]):
 
 
 def locate_objects(sample_table, radius):
-    """Locate the partitions (HEALPix order 5 pixels) each sample_table object is in.
+    """
+    Identify the HEALPix (order K=5) partitions intersecting a cone of radius
+    `radius` around each source. Helps determine which unWISE parquet files
+    must be loaded.
 
     Parameters
     ----------
-    sample_table : `~astropy.table.Table`
-        Table with the coordinates and journal reference labels of the sources
-    radius : astropy Quantity
-        radius for the cone search to determine which pixels overlap with each object
-
+    sample_table : astropy.table.Table
+        Table containing the source sample. The following columns must be present:
+            coord : astropy.coordinates.SkyCoord
+                Sky position of each source.
+            objectid : int
+                Unique identifier for each source in the sample.
+            label : str
+                Literature label for tracking source provenance.   
+    radius : astropy.units.Quantity
+        Angular radius used to determine which HEALPix pixels overlap each
+        source. Must be in angular units (e.g., arcsec or deg).
+        
     Returns
     -------
-    locations : pd.DataFrame
-        dataframe of location info including ["objectid", "coord.ra", "coord.dec", "label", "pixel"]
+    locations : pandas.DataFrame
+        Expanded dataframe containing one row per (object, HEALPix pixel)
+        combination. Columns include:
+
+            objectid : int  
+            label : str  
+            coord.ra : float (deg)  
+            coord.dec : float (deg)  
+            pixel : int
+                HEALPix pixel number at K=5 that overlaps the cone.
+
+    Notes
+    -----
+    * Each source may correspond to multiple HEALPix pixels.
```

Comment:
These bullet points may not render as intended, but again, we don't do rendering so it doesn't matter for now.